src/Elastic.Apm.NLog/ApmTraceIdLayoutRenderer.cs

```diff
@@ -15,8 +15,8 @@ public class ApmTraceIdLayoutRenderer : LayoutRenderer
 	{
 		protected override void Append(StringBuilder builder, LogEventInfo logEvent)
 		{
-			if (!Agent.IsConfigured) return;
-			if (Agent.Tracer.CurrentTransaction == null) return;
+			if (!Agent.IsConfigured || Agent.Tracer.CurrentTransaction == null)
```

Comment:
is `Agent.Tracer` guaranteed to always be not null if the agent is configured?

Reply:
Not sure, I will add a null check just in case.